### ISSUE TITLE
[EPIC] [Benefit Hubs] Improve "Ask questions" block on Benefits Hub landing pages, and the Support Service content type

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#7051]
## Background

Each Benefits Hub landing page (eg www.va.gov/health-care) has an accordion (open by default) called Ask Questions. 

<img width="351" alt="VA_Health_Care___Veterans_Affairs" src="https://user-images.githubusercontent.com/643678/143263817-2ac62d4a-1acb-43dd-befd-71d211f41917.png">

### User Story

As a Veteran, i want to know where to call or where to go if i have questions about my benefits. 

As a content manager of this content, i want to be able to create accessible, reusable content for Veterans, so that they can find the answers they need.  

### Problem statement

See this [mural](https://app.mural.co/t/vagov6717/m/vagov6717/1637759645192/acee5eb3704f92d2ca97ceffa14594adf833cbd3?sender=kevinwalsh8610) for some pain points, opportunities, and possible next steps. There's also a [Slack thread].

In short, the current setup does not support all the content design needs, and the FE has some edge case overrides for certain parts of the HTML, to make it read differently or to make it more accessible. 

### Affected users and stakeholders

* Managers of Benfits Hub Landing pages
* Possibly other editors in the future. Would this content live on Office pages too? 
* Public Websites team (FE owners)

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [ ] `Platform CMS Team`
- [x] `Sitewide CMS Team`
  - [x] `⭐️ Content ops`
  - [x] `⭐️ CMS experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product support`
  - [ ] `⭐️ User support`

**Comment by FranECross** [comment]
Will check with Jill about closing this ticket... It’s from Nov 2021, and the Mural it’s referring to won’t open, and there is no link to the Slack convo it references, so we don’t have insight into what improvements the person who created it was suggesting.

**Comment by FranECross** [comment]
Closing; will be revisited if this becomes a quarterly priority.
